tests/contrib/gevent/test_monkeypatch.py

```diff
@@ -24,7 +24,8 @@ def test_gevent_warning(monkeypatch):
 def test_gevent_auto_patching():
     import ddtrace
 
-    ddtrace.patch_all()
+    # Disable tracing sqlite3 as it is used by coverage
+    ddtrace.patch_all(sqlite3=False)
```

Comment:
👍 this has come up before and probably worth solving in general for CI

https://github.com/DataDog/dd-trace-py/blob/c437ad25a916f0abc8d2b44e7470cf659e7ade3e/tests/integration/test_integration.py#L774-L777